tests/layer_tests/common/layer_test_class.py

```diff
@@ -76,6 +76,11 @@ def _test(self, framework_model, ref_net, ie_device, precision, ir_version, temp
         #     (flag, resp) = ir.compare(ref_net)
         #     assert flag, '\n'.join(resp)
 
+        config = None
+        # GPU default execution precision is FP16, so if we want to check FP32 inference we need to set explicit precision hint
+        if ie_device == 'GPU' and precision == 'FP32':
+            config = {'INFERENCE_PRECISION_HINT' : 'f32'}
+
```

Comment:
I didn't understand how this config file will be used in the layer tests now. Please clarify